docs/src/_reference/cloud-cli.md

```diff
@@ -25,6 +25,41 @@ Logging out of Meltano Cloud invalidates your login token, and deletes the local
 meltano cloud logout
 ```
 
+## `project`
+
+The `project` command provides an interface for Meltano Cloud projects.
+
+The `list` subcommand shows all of the projects you have access to, and can use with other commands:
+
+```sh
+meltano cloud project list
+╭──────────┬───────────────────────────────┬──────────────────────────────────────────────────────────╮
+│  Active  │ Name                          │ Git Repository                                           │
+├──────────┼───────────────────────────────┼──────────────────────────────────────────────────────────┤
+│          │ Meltano Squared               │ https://github.com/meltano/squared.git                   │
+│          │ MDS-in-a-box                  │ https://github.com/aaronsteers/meltano-demo-in-a-box.git │
+╰──────────┴───────────────────────────────┴──────────────────────────────────────────────────────────╯
+```
+
+When you run the `login` command, if you only have a single project, it will be activated by default. Otherwise, you will need to run `meltano cloud project activate` to specify which Meltano Cloud project the other `meltano cloud` commands should operate on.
+
+```sh
+meltano cloud project activate 'Meltano Squared'
```

Comment:
I'm not fond of `switch` because it doesn't make sense when no project is active, and it feels a bit like something is being swapped.

@tayloramurphy @aaronsteers What do you think of `meltano cloud project use <project name>`?

Reply:
`use` could work! The other one that came to mind is `set-default` but I think that's a bit too heavy. @aaronsteers ?

Reply:
I didn't previously gave a string opinion on this, but you both make good points.

I can see `switch` being confusing if you haven't yet activated anything, and I can see an interpretation where `activate` would mean something like "deploy". 

I think "use" is nice: it's intuitive and hard to misinterpret.

Reply:
The `activate` command has been renamed to `use` in 033c58f